packages/dmn-js-decision-table/src/features/view-drd/ViewDrd.js

```diff
@@ -1,12 +1,13 @@
 import ViewDrdComponent from './components/ViewDrdComponent';
 
 export default class ViewDrd {
-  constructor(components, eventBus, injector, sheet) {
+  constructor(components, eventBus, injector, sheet, config) {

```

Comment:
Did you consider to also add this behavior to other `viewDRD` / `drillDown` components, as the issue description reveals?

cf. [`dmn-js-literal-expression@ViewDrd`](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-literal-expression/src/features/view-drd/ViewDrd.js)
cf. [`dmn-js-drd@DrillDown`](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-drd/src/features/drill-down/DrillDown.js#L127)

Reply:
How do I interact with the DrillDown in localhost to see its behavior?

Reply:
Sometimes it makes sense to mark one test case with `only` to ensure only this test is running.

> it.only(...)

Otherwise, karma would reuse the container for the next test case and you can't interact with it anymore.

Reply:
I tried using `only()` but it's not allowed, `"Unexpected exclusive mocha test"`. How do I fix this? I tried using `npm run dev -- dmn-js-drd --grep "should drill down"` as answered on stackoverflow and it still runs all the tests.

Reply:
The test itself is located in the `dmn-js-decision-table` package, therefore specifing the test with only and executing 

`npm run dev -- dmn-js-decision-table`

in the root folder of the project should work.

Reply:
Which test are you trying to run with "only"?

Reply:
It says `'karma' is not recognized as an internal or external command`, I tried npm install but didn't work.
PS: Because of my schedule and I'm new to this, if you find these PRs slow and not going anywhere we can close the PRs.

Reply:
I'm trying to test the drilldown part. I'm still not really sure, there are several tests in the DrillDownSpec.js.

Reply:
> Because of my schedule and I'm new to this, if you find these PRs slow and not going anywhere we can close the PRs.

We don't have to rush. Your PRs (this + https://github.com/bpmn-io/dmn-js/pull/588) tackle valid problems and basically, the solution sketch seems to be right for me.

However, it's up to you how and whether to continue with it. We can just try to support you and give you advice for specific questions you ask.